alws/app.py

```diff
@@ -31,6 +33,15 @@
 graceful_terminate_event = threading.Event()
 
 
+@app.middleware('http')
+async def permissions_middleware(request: Request, call_next):
```

Comment:
Where is this being used? I can't find any reference to it in the changes

Reply:
This is automatically applied, like added to application

Reply:
Got it, thanks for the clarification